backends-common/elasticsearch-v8/src/main/java/org/apache/james/backends/es/v8/ElasticSearchIndexer.java

```diff
@@ -84,7 +85,7 @@ public Mono<BulkResponse> update(List<UpdatedRepresentation> updatedDocumentPart
                 .index(aliasName.getValue())
                 .id(updatedDocumentPart.getId().asString())
                 .action(document -> document
-                    .withJson(new StringReader(updatedDocumentPart.getUpdatedDocumentPart())))
+                    .doc(new RawValue(updatedDocumentPart.getUpdatedDocumentPart())))
```

Comment:
I've been looking all afternoon for that... Rational being `withJson` is not working on bulkRequests atm => https://discuss.elastic.co/t/java-8-1-bulk-request/302423

...